control-plane/connect-inject/peering_acceptor_controller.go

```diff
@@ -278,7 +261,7 @@ func (r *PeeringAcceptorController) updateStatus(ctx context.Context, acceptor *
 			acceptor.Status.LatestPeeringVersion = pointerToUint64(peeringVersion)
 		}
 	}
-	err := r.Status().Update(ctx, acceptor)
+	err = r.Status().Update(ctx, acceptor)
 	if err != nil {
 		r.Log.Error(err, "failed to update PeeringAcceptor status", "name", acceptor.Name, "namespace", acceptor.Namespace)
 	}
```

Comment:
nit (in case you have time): can be one-lined. not a blocker.

Reply:
this one we can't update to one-liner because it'll change the scope of `err` and so we won't have the right error when we return on the below line (I already looked 😄 )

Reply:
ohno!! good point!! thanks so much for checking Iryna!!